src/DependencyInjection/SonataFormatterExtension.php

```diff
@@ -151,6 +157,18 @@ private function createEnvironment(
 
         $env->addMethodCall('addExtension', [new Reference(sprintf('sonata.formatter.twig.sandbox.%s', $code))]);
 
+        if (isset($bundles['SonataMediaBundle'])) {
+            $runtimeLoader = new Definition(ContainerRuntimeLoader::class, [
+                ServiceLocatorTagPass::register($container, [
+                    MediaRuntime::class => new Reference('sonata.media.twig.runtime'),
```

Comment:
I do not like to do it this way, because user could add its own extensions needing own custom runtime.

Reply:
Fixed.